openqaoa/rqaoa/rqaoa_results.py

```diff
@@ -55,4 +69,63 @@ def get_hamiltonian_step(self, i):
         """
         Returns the Hamiltonian of the i-th step of the RQAOA.
         """
-        return self.get_problem_step(i).hamiltonian
\ No newline at end of file
+        return self.get_problem_step(i).hamiltonian
+
+    def dumps(self, human:bool=False, as_string:bool=False, indent:int=4):
```

Comment:
Remvoe as_string; and create an  `as_dict()` funciton for the dictionary

Reply:
https://stackoverflow.com/questions/35282222/in-python-how-do-i-cast-a-class-object-to-a-dict#:~:text=not%20be%20desirable.-,Option,-2%3A